cpp/include/cudf/replace.hpp

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) 2018-2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "cudf.h"
+#include "types.hpp"
+
+// Forward declaration
+typedef struct CUstream_st* cudaStream_t;
+
+namespace cudf {
+
+namespace detail {
+
+/**
+ * @brief Replaces all null values in a column with corresponding values of another column
+ *
+ * The first column is expected to be a regular gdf_column. The second column
```

Comment:
What is a "regular" column? This documentation should be more precise.

Reply:
I am not sure what was meant by "regular" I reworded it.